target/linux/ramips/image/common-sercomm.mk

```diff
@@ -104,6 +129,11 @@ define Build/sercomm-payload
 	rm $@.pid
 endef
 
+define Build/sercomm-pid-setbit
+	printf 1 | dd seek=$$(($(word 1,$(1)))) of=$@ bs=1 conv=notrunc \
```

Comment:
Why take the first word? Can't it be replaced by $(1), assuming it is always called with a single argument?

Reply:
We can simplify this. Thanks. With `$(1)`:
```
cat log | grep "printf 1"
printf 1 | dd seek=$((0x11)) of=/home/rt-fl-1/openwrt/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7620/tmp/openwrt-ramips-mt7620-rostelecom_rt-fl-1-initramfs-factory.img bs=1 conv=notrunc 2>/dev/null
```